components/countdown/index.tsx

```diff
@@ -0,0 +1,65 @@
+import { useEffect, useState } from 'react';
+
+import Box from '@components/box';
+
+const getTargetDate = (): Date => {
+	const now = new Date();
+	const sundayNumber = 0;
+	const timezone = 'America/New_York';
+	const today = new Date(now);
+	today.setMilliseconds(0);
+	today.setSeconds(0);
+	today.setMinutes(0);
+	today.setHours(0);
+
+	const nextSunday = new Date(today);
+
+	do {
+		nextSunday.setDate(nextSunday.getDate() + 1);
+	} while (nextSunday.getDay() !== sundayNumber);
+
+	return new Date(nextSunday.toLocaleString('en-US', { timeZone: timezone }));
+};
+
+const getValues = (countDown: number): string[] => {
```

Comment:
It could be convenient to have a fixed size array as return type
```suggestion
const getValues = (countDown: number): [string, string, string, string] => {
```

Reply:
fixed in https://github.com/ardriveapp/inferno-leaderboard/pull/2/commits/cf627958c0eec81cb5cb6e6a9817b75b188c5975